docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/troubleshoot-azure-account-onboarding.adoc

```diff
@@ -141,7 +141,7 @@ config from cloud.resource where cloud.type = 'azure' AND api.name = 'azure-netw
 
 *Check that you have enabled flow logs on the NSGs*.
 
-.. Log in to the Azure portal, and select menu:Network{sp}Watcher[NSG Flow Logs] and verify that the status is *Enabled*.
+.. Log in to the Azure portal, and select *Network{sp}Watcher[NSG Flow Logs]* and verify that the status is *Enabled*.
```

Comment:
*Network{sp}Watcher[NSG Flow Logs]* must be changed to *Network Watcher > NSG Flow Logs*